itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationOnHDFSEncryptedZones.java

```diff
@@ -123,57 +122,24 @@ public void targetAndSourceHaveDifferentEncryptionZoneKeys() throws Throwable {
               put(HiveConf.ConfVars.REPLDIR.varname, primary.repldDir);
             }}, "test_key123");
 
-    List<String> dumpWithClause = Arrays.asList(
```

Comment:
seems like a testcase was removed; I wonder if this it not supported anymore ? ...and why are we removing this case in the scope of a hadoop upgrade?

Reply:
Same as above:
DistCp itself fails, It is running with hive.repl.add.raw.reserved.namespace and you can't copy if the key is not present on target cluster. Earlier I converted this to a failure case test, but then the next iteration fails which is without hive.repl.add.raw.reserved.namespace because the last load wasn't successful, so I kept the success case

I am not sure how it was working before, but with todat raw type the key should be there, earlier this test was also single instance, while fixing it became 2 instances....